src/lavinmq/exchange/default.cr

```diff
@@ -6,22 +6,28 @@ module LavinMQ
       "direct"
     end
 
-    def bind(destination, routing_key, headers = nil)
-      raise "Access refused"
+    def bindings_details : Iterator(BindingDetails)
+      Slice(BindingDetails).empty.each
```

Comment:
Monkey patch our code with https://github.com/crystal-lang/crystal/pull/15039 and uses that to get rid of an allocation?
```suggestion
   Iterator(BindingDetails).empty
```